test/hello_world/Dockerfile

```diff
@@ -1,9 +1,31 @@
 FROM ros:indigo-ros-base
 RUN apt-get update \
- && apt-get install -y \
+ && apt-get install -y build-essential qtbase5-dev \
       ros-indigo-ros-tutorials \
       ros-indigo-common-tutorials \
       vim \
       wget \
     && apt-get clean \
     && rm -rf /var/lib/apt/lists
+
+RUN mkdir /root/overlay_ws \
```

Comment:
Can you rename `/root/overlay_fs` to `/ros_ws`? Also, `ROS_WSPACE` could be lifted to before this `RUN` command.